### PR TITLE
Feature/services restaurants

### PR DESCRIPTION
**Pull request by Cristianjs93**
### Description
This pull request adds a CRUD feature for managing restaurants in the application.

### Changes
- Created API endpoints for creating, reading, updating, and deleting restaurant records.
- Implemented database models and schema for restaurants.

### CODE REVIEW

File: src/api/restaurants/restaurants.services.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      where: {
+         id
+      }
+   });
+
+   return restaurants;
+}
+
+export async function createRestaurant(input: Restaurants) {
+   // const hashedPassword = await hashPassword(input.password);
```
Comment (ht1204) [inline src/api/restaurants/restaurants.services.ts]: why is this commented?

File: src/api/restaurants/restaurants.services.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+   const data = {
+      ...input
+      //   password: hashedPassword
```
Comment (ht1204) [inline src/api/restaurants/restaurants.services.ts]: same here.

**Review by ht1204 — CHANGES_REQUESTED** [review]
Please, check the commented code and remove if it's not necessary. Thx.

**Review by ht1204 — APPROVED** [review]

**Review by r0odr1 — APPROVED** [review]
approve

**Pull request merged**